frontend/src/container/QueryBuilder/filters/GroupByFilter/GroupByFilter.tsx

```diff
@@ -75,8 +75,10 @@ export const GroupByFilter = memo(function GroupByFilter({
 										prefix: item.type || '',
 										condition: !item.isColumn,
 									}),
+									!item.isColumn && item.type ? item.type : '',
```

Comment:
pass the type on the same condition when the prefix was added so it gets removed only when it's added